appveyor.yml

```diff
@@ -1,7 +1,10 @@
 version: '{build}'
 environment:
   APPVEYOR_CACHE_ENTRY_ZIP_ARGS: "-t7z -m0=lzma -mx=9"
+image: Visual Studio 2017
 configuration:
+- MSVC_64
+- MSVC_32
```

Comment:
should be added to `allow_failures` below, then we can address the test failures in another PR.